docs/blog/posts/2024-01-24-test-pypi-groundhogs-day.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Resetting TestPyPI's state
+description: The database for test.pypi.org will be reset Friday February 2, 2024.
+authors:
+  - ewdurbin
+date: 2024-01-24
+tags:
+  - testpypi
+  - groundhogs
+---
+
+## What is happening?
+
+On February 2, 2024, and each following February 2nd,
+the state of [TestPyPI](https://test.pypi.org) will be reset completely.
+
+This includes **all** user accounts, projects, releases, and release files
+published to TestPyPI.
+
+In order to use TestPyPI all past and new users will need to
```

Comment:
Deleting all projects seems like a good idea, but deleting all users would make any automated usage of TestPyPI frustrating? You'd need to create a new account and tokens/TPs every year per project.

Many project CI's use TestPyPI to check if their package can be uploaded (due to package metadata requirements), if we're intending to remove that use-case we should recommend an alternative for folks in this post (Is `twine --check` sufficient these days?)